src/reynir_correct/errtokenizer.py

```diff
@@ -1789,6 +1847,65 @@ def suggest_word(
         token.add_corrected_meanings(m)
         return token
 
+    def add_ritmyndir_error(token: CorrectToken) -> CorrectToken:
+        """Add an error with corresponding correct value and details given info in Ritmyndir"""
+        # TODO At the moment the code assumes only one correct value is available in data
+        code = Ritmyndir.get_code(token.txt)
+        if code == "R001":
+            # Not an error
+            return token
+        if code == "SO-ÞGF4ÞF":
+            # Impersonal verbs that can have wrong subject case, not implicitly an error
+            return token
+        if code == "SN-TALA-GR":
+            # Unusual word forms, such as names with an article in the plural ('Jónarnir')
+            # Should not be marked as an error
+            return token
+        corrected = Ritmyndir.get_correct_form((token.txt))
+        if not corrected:
+            # No correct value available
+            return token
+        _, m = db.lookup_g(corrected, at_sentence_start=at_sentence_start)
+        text, details, refs = get_details(code, token.txt, corrected, m[0].stofn)
+        token.set_error(RitmyndirError(code, text, details, refs, token.txt, corrected))
+        if corrected not in {"in", "the", "for", "at"}:
+            # Exclude most common foreign stop words
+            token.txt = emulate_case(corrected, template=token.txt)
+        token.add_corrected_meanings(m)
+        return token
+
+    def get_details(
+        code: str, txt: str, correct: str, lemma: str
+    ) -> Tuple[str, str, List[str]]:
+        """Return short and detailed descriptions for the error category plus a link to grammar references where possible"""
+        # text is the short version, about the category and the error.
+        # details is the long version with references.
+        standref, cat, det = RitmyndirDetails.DICT[code]
+        references: List[str] = []
+        text = "{}: '{}' -> '{}'".format(cat, txt, correct)
+        details = det
+        if "{" in details:
```

Comment:
Not sure you really need this check; if the format spec is not present then a substitution is simply not made.

Reply:
Done